src/backend/joanie/core/factories.py

```diff
@@ -433,7 +433,9 @@ def courses(self, create, extracted, **kwargs):
             return
 
         for course in extracted:
-            CourseProductRelationFactory(product=self, course=course)
+            CourseProductRelationFactory(
+                product=self, course=course, organizations=course.organizations.all()
```

Comment:
Why ? In this way, you are declaring systematically authoring organizations as selling organizations. I'm pretty sure we don't want this behavior as default one.

Reply:
When i write the test, i had 48 organization and i feel it wrong.
I take time to find which factory create theses extra org and add this to avoid theses creations.

I can create a "seller organization" and use it in the right places.

Reply:
I looked at it and i've got no solution to give to this factory some "seller_organizations".
And I feel it wrong to have on new organization for each product.

Reply:
But from my point of view, there is no problem to generate new organization at this stage.